dts/arm/st/l4/stm32l4r5.dtsi

```diff
@@ -235,6 +235,20 @@
 			status = "disabled";
 			label= "OTGFS";
 		};
+
+		dmamux1: dmamux@40020800 {
+			compatible = "st,stm32-dmamux";
+			#dma-cells = <4>;
+			reg = <0x40020800 0x400>;
+			interrupts = <94 0>;
+			clocks = <&rcc STM32_CLOCK_BUS_AHB1 0x4>;
+			dma-channels = <14>;
+			dma-generators = <4>;
+			dma-requests= <26>;
```

Comment:
should be 89

Reply:
@shlomow what should be 89 ?

Reply:
@lowlander the `dma-requests`

Reply:
+ space mark ;-)  `dma-requests = <89>;`

Reply:
`<dma-requests>` is the Number of DMAMUX peripheral request inputs 
= 89 (STM32L4Rxxx and STM32L4Sxxx devices)
= 90 (STM32L4P5xx and STM32L4Q5xx devices)

(_26_ is the Number of DMAMUX request trigger inputs which is not relevant for this driver)

Reply:
This part will be "gone" from this PR after #30784 has been merged.